indexedCaps/solution.js

```diff
@@ -0,0 +1,8 @@
+function capitalize(s,arr){;
+  return s.split('')
+    .map((c, i) => {
```

Comment:
You could make this arrow function one line with ternary:

` .map( (c, i) => arr.includes(i) ? c.toUpperCase() : c )`